doc/install.txt

```diff
@@ -412,7 +412,7 @@ that we use.
 3) Install the ATLAS library. ATLAS is an open source optimized version of
 BLAS. You can install a precompiled version on most OSes, but if you're willing
 to invest the time, you can compile it to have a faster version (we have seen
-speed-ups of up to 3x, especialy on more recent computers, against the
+speed-ups of up to 3×, especially on more recent computers, against the
```

Comment:
@abergeron, @lamblin do you think this can cause problems? There is
problems if we use other charactere like é in the file NEWS.txt

On Mon, Jan 12, 2015 at 2:01 AM, Martin Thoma <notifications@github.com>
wrote:

> In doc/install.txt
> <https://github.com/Theano/Theano/pull/2391#discussion-diff-22776894>:
>
> > @@ -412,7 +412,7 @@ that we use.
> >  3) Install the ATLAS library. ATLAS is an open source optimized version of
> >  BLAS. You can install a precompiled version on most OSes, but if you're willing
> >  to invest the time, you can compile it to have a faster version (we have seen
> > -speed-ups of up to 3x, especialy on more recent computers, against the
> > +speed-ups of up to 3×, especially on more recent computers, against the
>
> I would use the times symbol (×) always when you want to say something
> like "three times".
>
> Why not put it between backquote to emphasis it?
> What do you mean with that? I did not want to emphasize it, I just wanted
> to use the typographical correct symbol. I think x is unfortunate, because
> usually x is used for variables (although everbody will understand it).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2391/files#r22776894>.
>